### PR TITLE
Voting system start for recipes

### PR DESCRIPTION
**Pull request by DavidFerri-PRJ**
Preliminary voting system
 Two things of note:  

1.  There is some redundant code that I can clean up at a later date.
2.  While a user can only vote on a recipe once, the vote up/down links remain.  I would like for them to be hidden after voting but I'm not sure how to do this at this time.

### CODE REVIEW

File: CookingCurator/App_Start/WebApiConfig.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+
+namespace CookingCurator
+{
+    public static class WebApiConfig
```
Comment (irrationalRock) [inline CookingCurator/App_Start/WebApiConfig.cs]: Do we need this class? What does it do?

File: CookingCurator/Web.config [hunk 2]
```diff
@@ -87,6 +74,214 @@
...
+				<assemblyIdentity name="System.Threading.Overlapped" publicKeyToken="B03F5F7F11D50A3A" culture="neutral" />
+				<bindingRedirect oldVersion="0.0.0.0-4.1.0.0" newVersion="4.1.0.0" />
+			</dependentAssembly>
+			<dependentAssembly>
+				<assemblyIdentity name="System.Threading" publicKeyToken="B03F5F7F11D50A3A" culture="neutral" />
+				<bindingRedirect oldVersion="0.0.0.0-4.0.11.0" newVersion="4.0.11.0" />
+			</dependentAssembly>
+			<dependentAssembly>
+				<assemblyIdentity name="System.Text.RegularExpressions" publicKeyToken="B03F5F7F11D50A3A" culture="neutral" />
+				<bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.1.1.0" />
```
Comment (hjpatel16) [inline CookingCurator/Web.config]: Is this changes necessary? I think its not needed

File: CookingCurator/Controllers/Manager.cs [hunk 3]
```diff
@@ -768,6 +768,56 @@ public bool ChangePW(RecoverViewModel resetPW)
...
+            }
+
+            query = "SELECT SUM(voting) FROM RECIPE_USERS WHERE recipe_ID = " + recipeId;
+            int sum = ds.Database.SqlQuery<int>(query).FirstOrDefault();
+
+            //Update recipe's overall rating
+            query = "UPDATE RECIPES SET rating = " + sum + " WHERE recipe_ID = " + recipeId;
+            ds.Database.ExecuteSqlCommand(query);
+            ds.SaveChanges();
+
```
Comment (irrationalRock) [inline CookingCurator/Controllers/Manager.cs]: I think your missing the end bracket here
